### PR TITLE
feat: roll Chromium to r564778

### LINKED ISSUE #452: roll Chromium to r564778
https://github.com/kblok/puppeteer/pull/5/commits/a0584689480c4e2102279969b515a837021f6f24

### LINKED ISSUE #482: test: drop PDF tests
https://github.com/kblok/puppeteer/pull/7/commits/c430138845a44e20bff88b49843569f119e08d3d

### PR DESCRIPTION
**Pull request by Meir017**
closes #452 
closes #482 

the rest of the changes are related to the worker class that doesn't exist yet...

### CODE REVIEW

**Comment by Meir017** [comment]
looks like a real fail...
image

**Comment by kblok** [comment]
It seems they got tired of that here https://github.com/kblok/puppeteer-sharp/issues/482

Though it's v1.6 you can go ahead and do that.

File: lib/PuppeteerSharp.Tests/PuppeteerSharp.Tests.csproj [hunk 1]
```diff
@@ -10,7 +10,6 @@
     <PackageReference Include="xunit" Version="2.3.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
     <PackageReference Include="SixLabors.ImageSharp" Version="1.0.0-beta0002" />
-    <PackageReference Include="PdfSharp" Version="1.50.4845-RC2a" NoWarn="NU1701" />
```
Comment (kblok) [inline lib/PuppeteerSharp.Tests/PuppeteerSharp.Tests.csproj]: Beautiful

File: lib/PuppeteerSharp.Tests/PageTests/PdfTests.cs [hunk 2]
```diff
@@ -1,138 +0,0 @@
...
-{
-    [Collection("PuppeteerLoaderFixture collection")]
-    public class PdfTests : PuppeteerPageBaseTest
-    {
-        public PdfTests(ITestOutputHelper output) : base(output)
-        {
-        }
-
-        [Fact]
-        public async Task ShouldBeAbleToSaveFile()
```
Comment (kblok) [inline lib/PuppeteerSharp.Tests/PageTests/PdfTests.cs]: We need to leave this test https://github.com/kblok/puppeteer/blob/c430138845a44e20bff88b49843569f119e08d3d/test/page.spec.js#L1363

**Comment by Meir017** [comment]
@kblok Done!

**Pull request merged**